air/src/options.rs

```diff
@@ -114,17 +115,17 @@ impl ProofOptions {
         assert!(fri_folding_factor >= 4, "FRI folding factor cannot be smaller than 4");
         assert!(fri_folding_factor <= 16, "FRI folding factor cannot be greater than 16");
 
-        assert!(fri_max_remainder_size.is_power_of_two(), "FRI max remainder size must be a power of 2");
-        assert!(fri_max_remainder_size >= 32, "FRI max remainder size cannot be smaller than 32");
-        assert!(fri_max_remainder_size <= 1024, "FRI max remainder size cannot be greater than 1024");
+        assert!(fri_remainder_size.is_power_of_two(), "FRI polynomial remainder size must be a power of 2");
+        assert!(fri_remainder_size >= 1, "FRI polynomial remainder size cannot be 0");
+        assert!(fri_remainder_size <= 32, "FRI polynomial remainder size cannot be greater than 32");
 
         ProofOptions {
             num_queries: num_queries as u8,
             blowup_factor: blowup_factor as u8,
             grinding_factor: grinding_factor as u8,
             field_extension,
             fri_folding_factor: fri_folding_factor as u8,
-            fri_max_remainder_size: fri_max_remainder_size.trailing_zeros() as u8,
```

Comment:
If we store remainder degree, this would be `log2(fri_max_remainder_degree + 1)`.